app/components/footer_component.html.erb

```diff
@@ -38,19 +38,14 @@
         </a>
       </div>
       <div class="site-footer-top__links-container">
-        <% helpers.navigation_resources.each do |resource| %>
-          <a href="<%= resource.path %>"><%= resource.title %></a>
-        <% end %>
-        <%= link_to "Find an event near you", events_path %>
-        <%= link_to "Sign up for updates", "/mailinglist/signup/name" %>
-        <%= link_to("Search", search_path) %>
       </div>
     </div>
     <div class="site-footer-bottom">
       <div class="site-footer-bottom__logo-container">
         <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education", size: "124x73" %>
```

Comment:
```suggestion
        <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education logo", size: "124x73" %>
```

I think we should change this alt text to be slightly more descriptive @garethwilliamsdesign (I know this is what it is in prod currently!)